frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx

```diff
@@ -23,7 +29,11 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
 }) => {
   const [notebookStates, loaded, error] = useProjectNotebookStates(project.metadata.name);
   const owner = getProjectOwner(project);
-
+  const navigate = useNavigate();
+  const [allowCreate] = useAccessReview({
```

Comment:
Could you take this to a custom hook & build the items for the Kebab from the data... no need to conditionally render two Kebab components